src/main/resources/data/plasmid/lang/fr_fr.json

```
@@ -0,0 +1,38 @@
+{
+  "text.plasmid.game.started.player": "%s a démarré le jeu !",
+  "text.plasmid.game.stopped.player": "%s a arrêté le jeu !",
+  "text.plasmid.game.stopped.error": "Une erreur inattendue est survenu lors de l'arrêt du jeu !",
+  "text.plasmid.game.list": "Jeux enregistrés :",
+  "text.plasmid.game.channel.create": "Canal '%s' créé",
+  "text.plasmid.game.channel.remove": "Canal '%s' supprimé",
+  "text.plasmid.game.channel.connect.entity": "'%1$s' connecté à '%2$s''",
+  "text.plasmid.game.channel.connect.block": "'%1$s' connecté au bloc avec pour position (%2$s; %3$s; %4$s)",
+  "text.plasmid.game.join": "%s a rejoint le lobby !",
+  "text.plasmid.game.join.error": "Une erreur inattendue est survenue lors de la connection au jeu !",
```

```suggestion
  "text.plasmid.game.join.error": "Une erreur inattendue est survenue lors de la connexion au jeu !",
```

connection is English

---

~~how do I always miss this one~~